.github/workflows/identity-e2e-tests.yml

```diff
@@ -34,9 +34,12 @@ concurrency:
   cancel-in-progress: true
   group: ${{ format('{0}-{1}', github.workflow, github.ref == 'refs/heads/main' && github.sha || github.ref) }}
 
+permissions:
+  contents: read # for git clone
+
 jobs:
   test:
-    runs-on: gcp-core-4-default
+    runs-on: ubuntu-latest
```

Comment:
I used `ubuntu-latest` as i noticed that it is much faster for the e2e tests when i used it in Optimize. Tasklist also uses  `ubuntu-latest` as well.